### PR TITLE
Addressing the issue dotnet/csharplang#1282

### PR DESCRIPTION
**Pull request by Unknown6656**
See the issue https://github.com/dotnet/csharplang/issues/1282 raised by @gafter.

### CODE REVIEW

**Comment by gafter** [comment]
We do not have a process for evolving the normative text of the specification at the moment. We expect the process will involve moving to the ECMA version of the specification as a starting point, with the integration of many Microsoft changes, and the ECMA committee (including Microsoft members) will probably handle the change requests from there. Whatever the process is, it won't be accepting PRs into this particular version of the specification, except perhaps for things like typographical errors introduced in the translation to markdown. This version of the spec is largely a placeholder for a process yet to be established.

I'm going to leave this PR open for now, but I think you should expect it to be closed.

@MadsTorgersen Hopefully you can say more at an appropriate time?

File: spec/expressions.md [hunk 1]
```diff
@@ -4821,6 +4821,7 @@ A constant expression must be the `null` literal or a value with one of  the fol
 *  References to `const` parameters or local variables
 *  Parenthesized sub-expressions, which are themselves constant expressions.
 *  Cast expressions, provided the target type is one of the types listed above.
+*  `null` literal conversions
```
Comment (gafter) [inline spec/expressions.md]: The name of the null literal conversion does not use a code font. I think this would be

> * Null literal conversions

Comment (Unknown6656) [reply spec/expressions.md]: Thank you, @gafter . I updated the line.

**Comment by Unknown6656** [comment]
Thank you very much, @gafter for your quick and elaborate response.

**Comment by AdamSpeight2008** [comment]
Changes should be okay if the specifications on here explicitly state it is non-ECMA standard.

**Comment by gafter** [comment]
@AdamSpeight2008 We have no intention of developing a language to diverge from ECMA. It is our intent to develop future versions of the specification jointly with ECMA. So taking changes to the spec of the core language that intentionally diverge from ECMA would be counterproductive.

**Pull request closed without merging**